utils/parse.js

```diff
@@ -107,6 +107,7 @@ exports.default = function parse(path, content, context) {
   if (context == null) { throw new Error('need context to parse properly'); }
 
   // ESLint in "flat" mode only sets context.languageOptions.parserOptions
+  console.log(context);
```

Comment:
still needed?

Reply:
I mean it was never needed but the underlying issue is still present, and where I think your input would be valuable - we're getting:

https://github.com/import-js/eslint-plugin-import/actions/runs/10626936020/job/29459577930?pr=2996#step:5:11751

```
    message: "Parse errors in imported module './tomdoc-deprecated': parserPath or languageOptions.parser is required! (undefined:undefined)",
```

If my memory is correct, I believe last time I looked into this is seemed like ESLint no longer provided the `parser` when you were using the default parser, which [seemed like it might be a bug](https://github.com/eslint/eslint/issues/17953) but then I couldn't reproduce it and didn't have the bandwidth to dig further.

I'd appreciate if you could deeper into this, at least if to confirm what this plugin is expecting behaviour wise from ESLint

Reply:
hmm - if it's not set, then presumably we could default it to espree in eslint 9?